herbert_core/utilities/classes/@object_lookup/object_array.m

```diff
@@ -25,5 +26,20 @@
 end
 
 % Get return argument
+if numel(varargin)==1
+    iarray = varargin{1};
+    if ~isscalar(iarray)
+        error('Index to original object array, ''iarray'', must be a scalar')
```

Comment:
There is an ADR recommending an identifier for errors in the form of 
`'(HORACE|HERBERT):(function_name|ClassName):error_identifier'`
https://github.com/pace-neutrons/Horace/blob/master/documentation/adr/0023-error-format.md

Reply:
Understood, and will follow the recommendations in future. I'd prefer not to edit this particular instance (and the possible one or two other new ones I introduced in this pull request) for the moment because since making this pull request Chris has asked me to reclone the repository following a bunch of changes that Alex, before I make any subsequent changes. This pull request would then straddle this process; I'd prefer to get this into master, and then edit later as there are going to be a lot of legacy error messages to edit in any case in the edited files. I also would like to clarify with you other aspects of the error handling as outlines in ADR 23 and 21.